ansible_collections/serverscom/sc_api/plugins/modules/sc_load_balancer_instance_l4.py

```diff
@@ -0,0 +1,524 @@
+      - Endpoint to use to connect to API.
+      - Do not change until specifically asked to do otherwise.
+  token:
+    type: str
+    required: true
+    description:
+      - Token to use.
+      - You can create token for your account in https://portal.servers.com
+        in Profile -> Public API section.
+  state:
+    description: Define the desired state of the load balancer instance.
+    required: true
+    type: str
+    choices: ["present", "absent"]
+  wait:
+    description:
+      - |
+        Maximum time in seconds to wait for the load balancer instance to reach
+        the desired state after an action (e.g. deletion)."
+      - "Default value is 600 seconds."
+    required: false
+    type: int
+    default: 600
+  update_interval:
+    description:
+      - "Interval in seconds between consecutive status checks during state transitions."
+      - "Default value is 5 seconds."
+    required: false
+    type: int
+    default: 5
+  id:
+    description:
+      - "Unique identifier of the load balancer."
+      - "For state 'present' (update), provide either 'id' or 'name' but not both."
+      - "For state 'absent', provide exactly one of 'id' or 'name'."
+    required: false
+    type: str
+  name:
+    description:
+      - "Name for the load balancer."
+      - "For creation (state 'present' without an 'id'), required."
+      - "For update, provide either 'id' or 'name' (but not both)."
+    required: false
+    type: str
+  location_id:
+    description: "Location ID where the load balancer will be created. (Required for creation)"
+    required: false
+    type: int
+  cluster_id:
+    description: >
+      Unique identifier of a dedicated Load Balancer cluster.
+      If omitted, a shared cluster is used.
+    required: false
+    type: str
+  store_logs:
+    description: "Enable log storage."
+    required: false
+    type: bool
+    default: false
+  store_logs_region_id:
+    description: "Cloud region ID for log storage."
+    required: false
+    type: int
+  new_external_ips_count:
+    description: "Number of new external IP addresses (update only)."
+    required: false
+    type: int
+  delete_external_ips:
+    description: "List of external IP addresses to delete (update only)."
+    required: false
+    type: list
+    elements: str
+  shared_cluster:
+    description: "If true, the load balancer will move to a shared cluster (update only)."
+    required: false
+    type: bool
+    default: true
+  vhost_zones:
+    description: "List of vhost zone objects defining forwarding rules. Required for creation."
+    type: list
+    elements: dict
+    suboptions:
+      id:
+        description: "Unique identifier of a zone. (1 to 255 characters)"
+        required: true
+        type: str
+      ports:
+        description: "Array of port numbers to apply the rule."
+        required: true
+        type: list
+        elements: int
+      udp:
+        description: "Enable UDP traffic balancing."
```

Comment:
Add that if not UDP, it will be TCP. (Obvious for us, may be reassuring for the new user).